src/components/FormTodo/FormTodo.tsx

```diff
@@ -0,0 +1,139 @@
+import { useState } from 'react';
+import { Todo, User } from '../../types/types';
+import usersFromServer from '../../api/users';
+import { getNewTodoId } from '../../utils/utils';
+import cn from 'classnames';
+
+type FormTodoProps = {
+  todos: Todo[];
+  addTodo: (todo: Todo) => void;
+};
+
+export const FormTodo: React.FC<FormTodoProps> = ({ todos, addTodo }) => {
+  const [users] = useState<User[]>(usersFromServer);
```

Comment:
why do you need this state? looks redundant

Reply:
The idea of using useState for users is coming to my mind. If we imagine that userFromServer is coming from the server, then useState can save this data after the first render, and subsequent renders will use this saved data in users. This is just my model. If we need to change users or add new ones, we already have the instrument for this. But in this case, it may be unnecessary.

Reply:
makes sense 👍